Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -436,6 +458,19 @@ open class BasicFormat: SyntaxRewriter {
       return trailingTrivia + Trivia(pieces: nextTokenLeadingWhitespace)
     }()
 
+    /// Whether the leading trivia of the token is folloed by a newline.
```

Comment:
Typo in `followed`

Reply:
Thanks. Found another two typos in the file as well.